bodhi/server/validators.py

```diff
@@ -454,7 +454,7 @@ def validate_acls(request, **kwargs):
             release = cache_release(request, build)
             if release is None:
                 return
-        elif 'update' in request.validated:
+        else:
```

Comment:
Is this change needed ? I would cautious about changing the logic in this validator :-)

Reply:
with the nature of the checker, there never seems to be a case where we would be able to get this far, and not have update in request.validated. I'll pull this change out though, but validators.py won't be 100% covered anymore...